etc/config/fortran.amazon.properties

```diff
@@ -0,0 +1,114 @@
+compilers=&gcc86:&cross
```

Comment:
This looks good. There's now a single (But recurring issue)
Group and compiler ids have to be unique, even between languages. My recommendation is that you add a `fo` before every id to differenciate it (That's how it's done in the C configs, for example)

Reply:
I did this for `fortran.default.properties` since this complained when I didn't (I used `gf` in this case). Overlooked it for `fortran.amazon.properties` and will fix it.